programmes-communautaires/partenariat.md

```diff
@@ -6,75 +6,79 @@ keywords:
- Pour cela, il faut:
- - Activer dans les paramètres de sécurité le niveau "Faible" : une adresse e-mail
- - Activer l'analyse des contenus médias des membres.
- - Paramétrer les notifications par défaut sur @mentions uniquement.
- - Retirer la permission à tout le monde de mentionner @everyone.
- - Accepter la [charte d'utilisation des serveurs commmunautaires](https://support.discord.com/hc/fr/articles/360035969312-Lignes-de-conduite-des-serveurs-communautaires) de discord.
- - Définir un salon des règles/conditions d'utilisation et un salon pour les annonces discord.
- 
- #### Serveurs communautaires
- 
- Voici un [lien](https://discord.fr/wiki/programmes-communautaires/outils-communautaires/serveur-communautaire) pour obtenir plus d'informations sur les serveurs communautaires.
- 
- #### Postuler
- Une fois le serveur communautaire activé, il faut aller dans l'onglet "Partner program".
- 
- ![Partner Program](https://i.discord.fr/86q.png)
- 
- Les conditions suivantes sont nécessaires pour postuler :
- - Avoir au moins 500 membres.
- - Avoir au moins 50 personnes actives (communiquant) sur le serveur par semaine.
- - Avoir au moins 100 visiteurs, qui ne participent pas forcément, mais qui "visitent le serveur".
- - Sur les utilisateurs "visitant" le serveur, il faut que 20 % ne quittent pas le serveur, et ce chaque semaine sur une durée totale de 8 semaines.
- 
- Les statistiques de serveurs sont disponibles ici : https://discord.com/developers/servers/
+#### 1. "Activer la communauté" dans les paramètres du serveur : {#activer-la-communaute}
+
+![Activer la communauté du serveur](https://github.com/volcanofr/dfr-wiki/assets/74511042/7d9fb8e8-32c7-422a-8ba6-6571c47c1535)
+
+
+Il faut tout d'abord configurer la communauté, Discord guide les utilisateurs tout au long de l'opération. Au final, l'utilisateur devra :
+
+- Activer le niveau de sécurité "Faible" au minimum (une adresse e-mail vérifiée est requise),
+- Activer l'analyse des contenus médias des membres,
+- Définir un salon des règles/chartes d'utilisation ainsi qu'un salon des mises à jour de Discord,
+- Paramétrer les notifications par défaut sur @Mentions uniquement,
+- Désactiver les permissions risquées pour `@everyone` (mentionner tout les rôles, gérer le serveur, les émojis, etc...),
+- Accepter les [Lignes de conduites](https://support.discord.com/hc/fr/articles/360035969312 'support.discord.com - Lignes de conduite des serveurs communautaires') ainsi que la [Charte d’utilisation de la communauté](https://discord.com/guidelines 'discord.com/guidelines') de Discord.
+
+[Cliquez ici](/wiki/gestion-serveur/outils-communautaires 'Discord.FR | Les serveurs communautaires') pour obtenir plus d'informations sur les serveurs communautaire.
+
+#### 2. Postuler au "Partner Program" : {#partner-program}
+
+![Partner Program](https://github.com/volcanofr/dfr-wiki/assets/74511042/28160845-08e8-4696-bd97-5c671ff61abf)
```

Comment:
Je n'ai pas importé l'image, puisqu'il serait peut-être préférable d'avoir un lien cliquable pour le bouton "Postuler".

Reply:
Comment ça ?
Il faudrait mettre un bouton renvoyant vers [discord.com/partners](https://discord.com/partners#how-to-apply 'discord.com/partners') tu penses ? Mais ils ne pourront pas directement postuler... Et si il faut vraiment mettre le lien, autant le mettre en haut de la page comme dans `certification.md`...

Reply:
Je mentionnais l'image dans mon message plus haut, je ne demandais pas à ce qu'un lien soit ajouté. Peut-être faudrait-il que le lien apparaisse comme cliquable, en sorte qu'il suffit de changer la couleur du bouton

Reply:
Je ne comprends pas ce que tu veux dire...
Tu veux ajouter un bouton, menant vers l'image, comme celui-là ? : 
![Bouton Docusaurus](https://github.com/discordfr/wiki/assets/74511042/07712d49-3988-4bfb-8860-26cf7d0a042b)

Reply:
(La discussion a été résolue sur le Discord)